retworkx/__init__.py

```diff
@@ -1 +1,5 @@
 from .retworkx import *
+
+
+class PyDAG(PyDiGraph):
```

Comment:
This wasn't checking for cycles previously, right?

Reply:
Right, in #28 we made the default behavior (for performance reasons) to not check for cycles by default. I probably would have preferred to make the `PyDAG` class just set `check_cycle` to true by default, so it's actuallya DAG. But, that would be a breaking change for any current 0.3.4 users out there so I didn't want to do it yet. (maybe for `0.5.0`, or `1.0.0`)